pages/api/unsubscribe.tsx

```diff
@@ -74,6 +74,14 @@ async function updateUnsub(uid: string, unsubKey: string) {
   }
 }
 
+/**
+ * Handles GET requests to the unsubscribe API.
+ * This endpoint is used to get the unsubscribe key for a specific user.
+ * The user is identified by their uid, which should be provided as a query parameter.
+ * @param {string} token - The admin token.
+ * @throws {InvalidBodyParamTypeError} If the uid query parameter is not a string.
+ * @returns {Promise<void>} A Promise that resolves when the unsubscribe key has been sent in the response.
+ */
```

Comment:
is this using this docs template? - https://www.jetbrains.com/help/webstorm/creating-jsdoc-comments.html#ws_js_jsdoc_typescript_syntax

Thanks for getting the docs started, I'll add them according to this style going forward

Reply:
Yeah, jsdocs. Makes it so you can document what does what, and it shows via intellisense (hover, etc.) That'd be awesome.